storagetransfer/storagetransfer_test.go

```diff
@@ -24,7 +24,10 @@ import (
 	"testing"
 	"time"
 
+	"github.com/aws/aws-sdk-go/service/sqs"
```

Comment:
I think this should be sorted under the non-core golang deps below (so after L39 I guess)